jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/cdi/ManagedBeanTest.java

```diff
@@ -234,6 +236,16 @@ public void producesAndDisposesObservesObservesAsync() throws Exception {
                 DiagnosticSeverity.Error, "jakarta-cdi", "RemoveDisposesOrConflictedAnnotations");
         
         assertJavaDiagnostics(diagnosticsParams, utils, d1, d2, d3, d4, d5, d6, d7, d8, d9, d10);
+        
+        
+        JakartaJavaCodeActionParams codeActionParams1 = createCodeActionParams(uri, d1);
+
+        TextEdit te1 = te(11, 4, 12, 4, "");
+        TextEdit te2 = te(12, 33, 13, 43, "");
+        CodeAction ca1 = ca(uri, "Remove @Inject", d1, te1);
+        CodeAction ca2 = ca(uri, "Remove the @Diposes modifier from this ", d1, te2);
+
+        assertJavaCodeAction(codeActionParams1, utils, ca1, ca2);
```

Comment:
The test is failing here as it expects 4 code actions, namely:
- "Remove the `@Observes` modifier from this "
- "Remove the `@ObservesAsync` from this "

Reply:
and how many code actions is it returning? What is the actual vs expected?

Reply:
Sorry for the confusion, I am expecting 2 code actions:
-  "Remove @Inject "
-  "Remove the @Disposes from this "

 But according to JUnit, the actual is 4.

Reply:
And are these the 4 code actions returned from JUnit?

- "Remove @Inject "
- "Remove the @Disposes from this "
- "Remove the @Observes modifier from this "
- "Remove the @ObservesAsync from this "

Reply:
Yes